test/i18n.js

```diff
@@ -3,99 +3,137 @@
-  }
-  catch (error) {
-    tape.ok(error);
-  }
-  tape.end();
-});
+describe("getLocalLang/setLocalLang", () => {
+  it("should force update on the local lang!", async() => {
+    await cacache.rm.entry(CACHE_PATH, "cli-lang");
 
-test("getLocalLang: force update on the local lang!", async(tape) => {
-  await cacache.rm.entry(CACHE_PATH, "cli-lang");
+    i18n.CONSTANTS.LANG_UPDATED = true;
+    await i18n.getLocalLang();
 
-  i18n.CONSTANTS.LANG_UPDATED = true;
-  await i18n.getLocalLang();
+    const { data } = await cacache.get(CACHE_PATH, "cli-lang");
 
-  tape.strictEqual(i18n.CONSTANTS.LANG_UPDATED, false);
-  tape.end();
-});
+    assert.strictEqual(data.toString(), "english");
+    assert.strictEqual(i18n.CONSTANTS.LANG_UPDATED, false);
+  });
 
-test("getLocalLang should return english by default", async(tape) => {
-  tape.strictEqual(await i18n.getLocalLang(), "english");
-  tape.end();
-});
+  it("should return english by default", async() => {
+    assert.strictEqual(await i18n.getLocalLang(), "english");
+  });
 
-test("setLocalLang to french", async(tape) => {
-  await i18n.setLocalLang("french");
-  tape.strictEqual(await i18n.getLocalLang(), "french");
-  tape.end();
-});
+  it("setLocalLang to french", async() => {
+    await i18n.setLocalLang("french");
+    assert.strictEqual(await i18n.getLocalLang(), "french");
+  });
 
-test("get languages", async(tape) => {
-  await i18n.setLocalLang("french");
-  tape.same((await i18n.getLanguages()).sort(), ["french", "english"].sort());
-  tape.end();
-});
+  it("should return all languages (with active one as first)", async() => {
+    await i18n.setLocalLang("french");
+    const languages = await i18n.getLanguages();
 
-test("Extend existing language", async(tape) => {
-  i18n.extend("french", { welcome: "Bienvenue" });
-  tape.deepEqual(await i18n.getToken("welcome"), "Bienvenue");
-  tape.end();
+    assert.strictEqual(languages.length, 2);
+    assert.strictEqual(languages[0], "french");
+    assert.strictEqual(languages[1], "english");
+  });
 });
 
-test("Add new language", async(tape) => {
-  i18n.extend("spanish", { welcome: "Bienvenido" });
-  await i18n.setLocalLang("spanish");
-  tape.deepEqual(await i18n.getToken("welcome"), "Bienvenido");
-  tape.end();
+describe("getToken", () => {
+  it("should throw an Error when called with a token that's not a string primitive", async() => {
+    await assert.rejects(
+      i18n.getToken(10),
+      {
+        name: "TypeError",
+        message: "token must be a string"
+      }
+    );
+  });
+
+  it("should throw an Error when called with an unknown token", async() => {
+    const expectedToken = "boo.foo";
+
+    await assert.rejects(
+      i18n.getToken(expectedToken),
+      {
+        name: "Error",
+        message: `Invalid i18n token -> ${expectedToken} for lang -> french`
+      }
+    );
+  });
+
+  it("should throw an Error when called with a token that's not a string primitive", async() => {
```

Comment:
The test description is wrong
```suggestion
  it("should throw an Error when called with an unknown language", async() => {
```